src/Watcher/IndexDocumentFactory.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\tao\elasticsearch\Watcher;
+
+use oat\tao\model\search\index\IndexDocumentBuilderInterface;
+
+class IndexDocumentFactory
+{
+    const AVAILABLE_INDEXERS = [
```

Comment:
I think that we have the same logic somewhere in the code. Could you check it please?

Reply:
I know what confuses you - it was a copy-paste of variable from one of your PR.
Same name, but different content inside. In any case - I'll remove it into smth like AVAILABLE_DOCUMENT_BUILDERS
Many thanks!